int32           selectedUAV
bool            BuildingTrajectoryReady
bool            ExecuteBuildingTrajectory